### DISCUSSION TITLE
ROKU IP Camera Viewer Pro

### DISCUSSION THREAD

**Discussion by RUHavingFun** [mrlt8/docker-wyze-bridge#1246] [Q&A]
Anyone have success getting video stream on ROKU?  I have setup "IP Camera Viewer Pro".  Was unable to get RTSP to work (seems an H.264 issue).  Was also unable to get HLS working either.  UI is very clunky.

I was finally successful using the Snapshots, so I am OK with this, just is just a bit jumpy as I get the snapshots every 5s to limit the pulls.

Would love to know if anyone else has had success.

**Comment by mrlt8** [comment]
Roku cam is on my todo list.

**Reply by mikeofree** [reply]
any updates?
